modules/reactor/include/meshgenerators/PeripheralTriangleMeshGenerator.h

```diff
@@ -27,28 +23,22 @@ class PeripheralTriangleMeshGenerator : public MeshGenerator
   std::unique_ptr<MeshBase> generate() override;
 
 protected:
-  /// Subdomain ID of created tri elements
-  const SubdomainID _tri_subdomain_id;
-  /// inner boundary mesh
-  std::unique_ptr<MeshBase> & _inner_boundary_mesh;
-  /// inner boundary id
-  boundary_id_type _inner_boundary_id;
-  /// outer circle boundary radius
-  const Real _outer_circle_radius;
-  /// Number of segments in the outer circle boundary
-  const unsigned int _outer_circle_num_segments;
-  /// The boundary id for the generated mesh outer boundary.
-  const unsigned int _outer_boundary_id;
-  /// Radii of concentric Steiner circles
-  const std::vector<Real> _extra_circle_radii;
```

Comment:
Does the `XYDelaunay` refinement completely obsolete the ability to add extra Steiner point circles?  We'll get you down to the same mesh size, probably with a smoother mesh if you have a convoluted triangulated domain, but we won't get you the same symmetry.

Reply:
I suppose we could find a way to overlap them and allow users to input extra Steiner point circles that could then be further refined. I think that would probably be extending `XYDelaunayGenerator` to accept a list of Steiner points that could be applied along with the area refinement, and I could build that list to provide in PTMG using the existing functionality. 
But the feeling I got from the users was they didn't particularly like putting in the extra circles and were only doing it because it was better than nothing, and I think would generally prefer to use the automatic refinement. 
@eshemon what do you think?

Reply:
I have to say I love how much code you were able to remove via this PR, and I won't fight too hard if nobody thinks it's worth putting back.

And to be fair, even with Steiner point circles there's no guarantee of perfect symmetry; trapezoids have to be divided along one diagonal or another and with poly2tri (with floating-point arithmetic in general?) the question of which division takes place is basically a coin flip.